Automattic/themes-workspace
# Add github actions to deploy to staging site

This PR automates theme deployment to the [staging site](https://botanytheme.mystagingwebsite.com/). 

Whenever commits are made / merged to master (or this branch), a github action will run to upload the `varya` directory via FTP to the themes directory of the staging site.

To test this, you can:
1. make a change to this branch (e.g. add a `test.php` file)
2. commit it
3. verify that it is uploaded to the correct directory on the staging site
